firebase-appdistribution/src/main/java/com/google/firebase/appdistribution/FirebaseAppDistribution.java

```diff
@@ -64,6 +64,8 @@
   private Task<AppDistributionRelease> cachedCheckForNewReleaseTask;
   private AlertDialog updateDialog;
   private AlertDialog signInConfirmationDialog;
+  private boolean remakeUpdateDialog = false;
```

Comment:
Is there any risk of multiple threads reading these values? do these need to be guarded by locks?

Reply:
That is a good question. This is only changed in the onDestroyed and the onCreate. I am unsure if other threads will be able to change the value since it is tied to the activity.